filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -1,141 +1,238 @@
+*deb and rpm:*
 +
-This command takes care of configuring Filebeat and loading the ingest node
-pipelines and other configuration settings required to parse the log files.
+["source","sh",subs="attributes"]
+----
+{beatname_lc} setup -e
+----
 +
-To run more than one module, specify a comma-separated list of modules. For
-example:
+*mac:*
 +
-[source,shell]
-----------------------------------------------------------------------
-./filebeat -e --modules system,nginx,mysql
-----------------------------------------------------------------------
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} setup -e
+----
 +
-When you start Filebeat, you should see messages indicating that Filebeat
-has started harvesters for all enabled modules. For example:
+*win:*
 +
-[source,shell]
-----------------------------------------------------------------------
-2017/08/16 23:39:15.414375 harvester.go:206: INFO Harvester started for file: /var/log/displaypolicyd.stdout.log
-----------------------------------------------------------------------
+["source","sh",subs="attributes"]
+----
+PS > .{backslash}{beatname_lc}.exe setup -e
+----
++
+The <<setup-command,`setup`>> command loads the recommended index template for
+writing to {es} and deploys the sample dashboards for visualizing the
+data in {kib}. This is a one-time setup step. 
 +
-If you don't see this message for each log file that needs to be read,
-see <<setting-variables>> to find out how to set the path the files.
+The `-e` flag is optional and sends output to standard error instead of syslog.
 
-NOTE: Depending on how you've installed Filebeat, you might see errors
-related to file ownership or permissions when you try to run Filebeat modules.
+. Run {beatname_uc}:
++
+*deb and rpm:*
++
+["source","sh",subs="attributes"]
+----
+service {beatname_lc} start
+----
++
+*mac:*
++
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} -e
+----
++
+*win:*
++
+["source","sh",subs="attributes"]
+----
+PS > Start-Service {beatname_lc}
+----
++
+If the module is configured correctly, you'll see
+`INFO Harvester started` messages for each file specified in the config.
++
+If your logs aren't in the default location, see <<setting-variables>> to find
+out how to set the path the files, then run {beatname_uc} after you've
+configured the module.
++
+NOTE: Depending on how you've installed {beatname_uc}, you might see errors
+related to file ownership or permissions when you try to run {beatname_uc} modules.
 See {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_ if you encounter errors related to file
-ownership or permissions.
-
-This getting started guide uses the `--modules` flag to enable modules
-at the command line when you run Filebeat. In a production environment, you'll
-probably want to use the configs in the `modules.d` directory instead. See
-<<configuration-filebeat-modules>> for more information.
+ownership or permissions. 
```

Comment:
Both the note and the paragraph are talking about `errors related to file ownership or permissions`. May this sound repetitive? I'm not sure here, so it's your call :)